.github/workflows/release.yml

```diff
@@ -35,14 +35,43 @@ jobs:
           set -euo pipefail
 
           while IFS="" read -r -d "" buildpack_toml_path; do
-            buildpack_id="$(yj -t < "${buildpack_toml_path}" | jq -r .buildpack.id)"
-            buildpack_version="$(yj -t < "${buildpack_toml_path}" | jq -r .buildpack.version)"
-            buildpack_docker_repository="$(yj -t < "${buildpack_toml_path}" | jq -r .metadata.release.docker.repository)"
+            buildpack_id="$(yj -t <"${buildpack_toml_path}" | jq -r .buildpack.id)"
```

Comment:
What do you think about moving this script to its own file, so shellcheck can lint it + the Circle CI config is not quite so verbose?

Reply:
I would love to! I did not research how this would be possible with GitHub Actions yet. I'll keep it in mind. :)